DHT plugin fix 'NaN' issue

krikk opened the pull request:
issue https://github.com/letscontrolit/ESPEasy/issues/284 seems valid to me, implemented...

reported be @linker3000

beicnet commented:
Tested also in my custom build and its working just fine!

psy0rz commented:
@krikk  is it possible you broke something? 

https://www.letscontrolit.com/forum/viewtopic.php?f=18&t=2257&start=160#p16315

Updated today to DEV9, and DHT22 shows 'nan' now in device list. Reverted to DEV8 and all is ok again.  :( Hope we don't have to wait over a month for the next update to fix that....

remb0 commented:
same issue as psy0rz.

krikk commented:
just tested with my dht22, everything working fine...

![image](https://cloud.githubusercontent.com/assets/5165445/26558364/58a34516-44a8-11e7-9c34-4ed2ff20629d.png)

jarko75uk commented:
Hi, just flashed and seems to be the same, nan shows by device dht22. Any ideas?

krikk commented:
even tested with 2 sensors:

![image](https://cloud.githubusercontent.com/assets/5165445/26558641/b05569c2-44aa-11e7-8704-a965b05ff841.png)

krikk commented:
so to make sure i just tested with the firmware downloaded from github... and i can reproduce it... but this does only happen with the firmware from the release zip...

if i compile the latest source by my self it does work as expected

krikk commented:
@psy0rz i have no clue why it does work in my selfcompiled firmware but not in our release firmware...

krikk commented:
reverted the change,  https://github.com/letscontrolit/ESPEasy/pull/320

...but i still have no clue why this does not work at all in the release build... in my self compiled firmware it does work with and without the fix, but i get occasional "no readings" with the fix, so i reverted it...
